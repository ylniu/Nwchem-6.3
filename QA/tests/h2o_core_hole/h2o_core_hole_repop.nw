
echo
start h2o_core_hole_repop

memory 1000 mb

geometry units au
 O 0       0        0
 H 0       1.430   -1.107
 H 0      -1.430   -1.107
end


basis
  O library 6-31g*
  H library 6-31g*
end

# read in core-hole optimized vectors
# repopulate occupancy and orthonormalize virtuals with filled

dft
 xc beckehandh
 vectors input core-hole.movecs output repop.movecs
 noscf
end
task dft
