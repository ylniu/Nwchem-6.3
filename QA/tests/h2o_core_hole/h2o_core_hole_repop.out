 argument  1 = h2o_core_hole_repop.nw



============================== echo of input deck ==============================

echo
start h2o_core_hole_repop

memory 1000 mb

geometry units au
 O 0       0        0
 H 0       1.430   -1.107
 H 0      -1.430   -1.107
end


basis
  O library 6-31g*
  H library 6-31g*
end

# read in core-hole optimized vectors
# repopulate occupancy and orthonormalize virtuals with filled

dft
 xc beckehandh
 vectors input core-hole.movecs output repop.movecs
 noscf
end
task dft
================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 6.1.1
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = orion
    program         = ../../../bin/LINUX64/nwchem
    date            = Mon Jun  4 15:57:07 2012

    compiled        = Mon_Jun_04_15:53:01_2012
    source          = /home/niri/nwchem/nwchem-trunk
    nwchem branch   = Development
    nwchem revision = 22009:22553M
    ga revision     = 9973
    input           = h2o_core_hole_repop.nw
    prefix          = h2o_core_hole_repop.
    data base       = ./h2o_core_hole_repop.db
    status          = startup
    nproc           =        4
    time left       =     -1s



           Memory information
           ------------------

    heap     =   32768001 doubles =    250.0 Mbytes
    stack    =   32768001 doubles =    250.0 Mbytes
    global   =   65536000 doubles =    500.0 Mbytes (distinct from heap & stack)
    total    =  131072002 doubles =   1000.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


 C2V symmetry detected

          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000     0.22140000
    2 H                    1.0000    -1.43000000     0.00000000    -0.88560000
    3 H                    1.0000     1.43000000     0.00000000    -0.88560000

      Atomic Mass 
      ----------- 

      O                 15.994910
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)       9.1971984402

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             C2v       
 Group number             16
 Group order               4
 No. of unique centers     2

      Symmetry unique atoms

     1    2



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.95697
    2 Stretch                  1     3                       0.95697
    3 Bend                     2     1     3               104.51124


            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000     0.11715984
 H                    -0.75672347     0.00000000    -0.46863937
 H                     0.75672347     0.00000000    -0.46863937

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.80841  |     1.80841
    3 H                |   1 O                |     1.80841  |     1.80841
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   104.51
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/niri/nwchem/nwchem-trunk/src/basis/libraries/>
  
                      Basis "ao basis" -> "" (cartesian)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.48467170E+03  0.001831
  1 S  8.25234950E+02  0.013950
  1 S  1.88046960E+02  0.068445
  1 S  5.29645000E+01  0.232714
  1 S  1.68975700E+01  0.470193
  1 S  5.79963530E+00  0.358521

  2 S  1.55396160E+01 -0.110778
  2 S  3.59993360E+00 -0.148026
  2 S  1.01376180E+00  1.130767

  3 P  1.55396160E+01  0.070874
  3 P  3.59993360E+00  0.339753
  3 P  1.01376180E+00  0.727159

  4 S  2.70005800E-01  1.000000

  5 P  2.70005800E-01  1.000000

  6 D  8.00000000E-01  1.000000

  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757

  2 S  1.61277800E-01  1.000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31g*                  6       15   3s2p1d
 H                           6-31g*                  2        2   2s



                                 NWChem DFT Module
                                 -----------------


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     3
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  30
          AO basis - number of functions:    19
                     number of shells:    10
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
           Becke half-and-half Method XC Potential
                     Hartree-Fock (Exact) Exchange  0.500          
                        Slater Exchange Functional  0.500 local    
            Perdew 1991 LDA Correlation Functional  0.500 local    

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          O                   0.60       49           5.0       434
          H                   0.35       45           6.0       434
          Grid pruning is: on 
          Number of quadrature shells:    94
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


 Loading old vectors from job with title :




      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------

  Numbering of irreducible representations: 

     1 a1          2 a2          3 b1          4 b2      

  Orbital symmetries:

     1 a1          2 a1          3 b1          4 a1          5 b2      
     6 a1          7 b1          8 a1          9 b2         10 b1      
    11 a1         12 b1         13 a1         14 a1         15 a2      

   Time after variat. SCF:      0.1
   Time prior to 1st pass:      0.1

 #quartets = 1.009D+03 #integrals = 5.756D+03 #direct =  0.0% #cached =100.0%


 Integral file          = ./h2o_core_hole_repop.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  10690
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


 Grid_pts file          = ./h2o_core_hole_repop.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =      5        Max. recs in file   =     57016



         Total DFT energy =      -75.273031125809
      One electron energy =     -128.366869870935
           Coulomb energy =       53.610711542264
    Exchange-Corr. energy =       -9.714071237336
 Nuclear repulsion energy =        9.197198440198

 Numeric. integr. density =       10.000001572160

     Total iterative time =      0.0s



                  Occupations of the irreducible representations
                  ----------------------------------------------

                     irrep           alpha         beta
                     --------     --------     --------
                     a1                3.0          3.0
                     a2                0.0          0.0
                     b1                1.0          1.0
                     b2                1.0          1.0


                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-2.158382D+01  Symmetry=a1
              MO Center= -2.1D-18, -8.1D-21,  1.2D-01, r^2= 1.4D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      1.002173  1 O  s          

 Vector    2  Occ=2.000000D+00  E=-1.725827D+00  Symmetry=a1
              MO Center= -3.5D-18,  3.3D-35, -5.4D-02, r^2= 4.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.583390  1 O  s                  6      0.500775  1 O  s          
     1     -0.223340  1 O  s          

 Vector    3  Occ=2.000000D+00  E=-1.162534D+00  Symmetry=b1
              MO Center= -6.9D-18, -4.2D-18, -2.6D-02, r^2= 5.4D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.690662  1 O  px                 7      0.220921  1 O  px         
    16     -0.192268  2 H  s                 18      0.192268  3 H  s          

 Vector    4  Occ=2.000000D+00  E=-1.040538D+00  Symmetry=a1
              MO Center=  6.2D-17,  8.4D-18,  2.1D-01, r^2= 4.6D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.730772  1 O  pz                 6      0.265233  1 O  s          
     9      0.254140  1 O  pz                 2      0.194258  1 O  s          

 Vector    5  Occ=2.000000D+00  E=-9.826229D-01  Symmetry=b2
              MO Center= -4.4D-17,  6.1D-18,  9.8D-02, r^2= 4.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.802784  1 O  py                 8      0.315932  1 O  py         

 Vector    6  Occ=0.000000D+00  E=-7.398055D-01  Symmetry=a1
              MO Center=  0.0D+00,  8.5D-19, -6.6D-01, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.084923  1 O  s                 17     -0.872899  2 H  s          
    19     -0.872899  3 H  s                  9     -0.296420  1 O  pz         
     5     -0.293540  1 O  pz                 2      0.207326  1 O  s          
    16     -0.175352  2 H  s                 18     -0.175352  3 H  s          

 Vector    7  Occ=0.000000D+00  E=-6.668602D-01  Symmetry=b1
              MO Center=  3.3D-16,  6.6D-20, -5.6D-01, r^2= 2.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17     -1.105450  2 H  s                 19      1.105450  3 H  s          
     7     -0.511061  1 O  px                 3     -0.452112  1 O  px         
    16     -0.218535  2 H  s                 18      0.218535  3 H  s          

 Vector    8  Occ=0.000000D+00  E=-4.993664D-02  Symmetry=a1
              MO Center= -2.8D-15,  3.9D-17,  3.5D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.988522  1 O  pz                 5     -0.811711  1 O  pz         
    17      0.458125  2 H  s                 19      0.458125  3 H  s          
    16     -0.310718  2 H  s                 18     -0.310718  3 H  s          

 Vector    9  Occ=0.000000D+00  E=-4.969660D-02  Symmetry=b2
              MO Center=  1.0D-16,  2.5D-19,  1.1D-01, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.110809  1 O  py                 4     -0.831613  1 O  py         

 Vector   10  Occ=0.000000D+00  E=-1.514540D-02  Symmetry=b1
              MO Center=  3.4D-15, -1.2D-17, -2.8D-01, r^2= 1.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17     -0.898990  2 H  s                 19      0.898990  3 H  s          
    16      0.810561  2 H  s                 18     -0.810561  3 H  s          
    12      0.367989  1 O  dxz                3      0.268539  1 O  px         

 Vector   11  Occ=0.000000D+00  E= 8.152939D-02  Symmetry=a1
              MO Center= -1.0D-15,  2.9D-19, -2.7D-01, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.207105  1 O  s                 16      0.758035  2 H  s          
    18      0.758035  3 H  s                  2     -0.743697  1 O  s          
    17     -0.635231  2 H  s                 19     -0.635231  3 H  s          
     9      0.504324  1 O  pz                13     -0.299370  1 O  dyy        
    15     -0.192193  1 O  dzz        

 Vector   12  Occ=0.000000D+00  E= 1.522361D-01  Symmetry=b1
              MO Center= -2.6D-15, -2.9D-17,  1.6D-01, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.799227  1 O  px                17      0.987184  2 H  s          
    19     -0.987184  3 H  s                  3     -0.835065  1 O  px         

 Vector   13  Occ=0.000000D+00  E= 3.642551D-01  Symmetry=a1
              MO Center=  2.6D-15,  5.4D-17, -3.4D-01, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      3.656614  1 O  s                  2     -1.477084  1 O  s          
     9     -1.003530  1 O  pz                17     -0.792860  2 H  s          
    19     -0.792860  3 H  s                 10     -0.647798  1 O  dxx        
    16     -0.377495  2 H  s                 18     -0.377495  3 H  s          
    15     -0.345492  1 O  dzz                5      0.260730  1 O  pz         

 Vector   14  Occ=0.000000D+00  E= 9.051363D-01  Symmetry=a1
              MO Center=  1.1D-16, -5.5D-17,  1.8D-01, r^2= 6.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      1.008292  1 O  dzz               10     -0.572156  1 O  dxx        
    13     -0.369675  1 O  dyy                6     -0.288168  1 O  s          
     9      0.172734  1 O  pz         

 Vector   15  Occ=0.000000D+00  E= 9.173171D-01  Symmetry=a2
              MO Center= -6.4D-16,  1.1D-16,  1.2D-01, r^2= 6.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.732051  1 O  dxy        


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.09751021

 moments of inertia (a.u.)
 ------------------
           2.193637940261           0.000000000000           0.000000000000
           0.000000000000           6.315440625261           0.000000000000
           0.000000000000           0.000000000000           4.121802685000

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      0.000000     -5.000000     -5.000000     10.000000

     1   1 0 0      0.000000      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000      0.000000
     1   0 0 1     -1.306738     -0.653369     -0.653369      0.000000

     2   2 0 0     -0.955370     -2.522585     -2.522585      4.089800
     2   1 1 0      0.000000      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000      0.000000
     2   0 2 0     -3.904363     -1.952182     -1.952182      0.000000
     2   0 1 1      0.000000      0.000000      0.000000      0.000000
     2   0 0 2     -2.750696     -2.355707     -2.355707      1.960718


 Parallel integral file used       4 records with       0 large values


 Task  times  cpu:        0.1s     wall:        1.3s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:   36       36      356      137      118        0        0        0     
number of processes/call 1.52e+00 1.26e+00 1.15e+00 0.00e+00 0.00e+00
bytes total:             1.75e+05 5.40e+04 7.47e+04 0.00e+00 0.00e+00 0.00e+00
bytes remote:            5.32e+04 1.35e+04 1.90e+04 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 30440 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        24	        49
	current total bytes		         0	         0
	maximum total bytes		   1552760	  18337272
	maximum total K-bytes		      1553	     18338
	maximum total M-bytes		         2	        19


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
  E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
          M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus, J. Hammond,
    J. Autschbach, F. Aquino, S. Hirata, M. T. Hackler, J. Mullin, P. Nichols,
    R. Peverati, Y. Zhao, P.-D. Fan, R. J. Harrison, M. Dupuis, D. M. A. Smith,
    J. Nieplocha, V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen,
      M. Swart, Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby,
        E. Brown, G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
   X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann,
          G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                                A. Wong, Z. Zhang.

 Total times  cpu:        0.1s     wall:        2.6s
